docs/FIELD_ACCESS.md

```diff
@@ -0,0 +1,456 @@
+let len = get_tx_nested_field(
+    locator.buffer.as_ptr(),
+    locator.cur_buffer_index,
+    buffer.as_mut_ptr(),
+    buffer.len()
+)?;
+```
+
+### Accessing Signer Fields
+
+```rust
+// Access second signer's account
+let mut locator = Locator::new();
+locator.pack(sfield::Signers);
+locator.pack(1);  // Second signer (0-indexed)
+locator.pack(sfield::Account);
+
+let mut account = [0u8; 20];
+let len = get_tx_nested_field(
+    locator.buffer.as_ptr(),
+    locator.cur_buffer_index,
+    account.as_mut_ptr(),
+    account.len()
+)?;
+```
+
+### Accessing Oracle Data
+
+```rust
+// Access price from oracle document
+let mut locator = Locator::new();
+locator.pack(sfield::PriceDataSeries);
+locator.pack(0);
+locator.pack(sfield::AssetPrice);
+
+let mut price_buf = [0u8; 8];
+let len = get_ledger_obj_nested_field(
+    oracle_slot,
+    locator.buffer.as_ptr(),
+    locator.cur_buffer_index,
+    price_buf.as_mut_ptr(),
+    price_buf.len()
+)?;
+```
+
+### Iterating Arrays
+
+```rust
+// Get array length
+let memo_count = get_tx_array_len(sfield::Memos)?;
+
+// Process each memo
+for i in 0..memo_count {
+    // Access MemoType
+    let mut type_locator = Locator::new();
+    type_locator.pack(sfield::Memos);
+    type_locator.pack(i as i32);
+    type_locator.pack(sfield::MemoType);
+    
+    // Access MemoData
+    let mut data_locator = Locator::new();
+    data_locator.pack(sfield::Memos);
+    data_locator.pack(i as i32);
+    data_locator.pack(sfield::MemoData);
+    
+    // Process memo...
+}
+```
+
+## Field Code Reference
+
+### Common Transaction Fields
+
+| Field | Code | Type | Size |
+|-------|------|------|------|
+| Account | 524289 | AccountID | 20 |
+| TransactionType | 65538 | UInt16 | 2 |
+| Fee | 393224 | Amount | 8 |
+| Sequence | 131076 | UInt32 | 4 |
+| Flags | 131074 | UInt32 | 4 |
+| SourceTag | 131075 | UInt32 | 4 |
+| DestinationTag | 131086 | UInt32 | 4 |
+| SigningPubKey | 458755 | Blob | 33 |
+| TxnSignature | 458756 | Blob | Variable |
+
+### Escrow Fields
+
+| Field | Code | Type | Size |
+|-------|------|------|------|
+| Owner | 524290 | AccountID | 20 |
+| Destination | 524291 | AccountID | 20 |
+| Amount | 393217 | Amount | 8 |
+| Condition | 458769 | Blob | 32 |
+| CancelAfter | 262156 | UInt32 | 4 |
+| FinishAfter | 262157 | UInt32 | 4 |
+| SourceTag | 196624 | UInt32 | 4 |
+| DestinationTag | 196622 | UInt32 | 4 |
+
+### Array Fields
+
+| Field | Code | Type |
+|-------|------|------|
+| Memos | 983049 | Array of Memo |
+| Signers | 983043 | Array of Signer |
+| SignerEntries | 720900 | Array of SignerEntry |
+| Paths | 65537 | Array of Path |
+
+### Nested Object Fields
+
+| Parent | Field | Code | Type |
+|--------|-------|------|------|
+| Memo | MemoType | 458764 | Blob |
+| Memo | MemoData | 458765 | Blob |
+| Memo | MemoFormat | 458766 | Blob |
+| Signer | Account | 524291 | AccountID |
+| Signer | TxnSignature | 524292 | Blob |
+| Signer | SigningPubKey | 524293 | Blob |
+
```

Comment:
consider add a sentence about user should use named sfields such as `sfield::MemoType` instead of int values.